app/jobs/get_next_a_i_message_job.rb

```diff
@@ -0,0 +1,29 @@
+class GetNextAIMessageJob < ApplicationJob
+  def perform(conversation_id, assistant_id)
+    puts "GetNextAIMessageJob.perform(#{conversation_id}, #{assistant_id})"
+
+    @conversation = Conversation.find conversation_id
+    @assistant = Assistant.find assistant_id
+
+    @new_message = @conversation.messages.create! role: :assistant, content_text: "", assistant: @conversation.assistant
+    @new_message.broadcast_append_to @conversation
```

Comment:
Sometimes the job is slow to be picked up. If we moved these two lines to be in the controller then we'd get an instant stub of the assistant response even if it takes awhile to process. But that was out of scope here so I didn't do yet.

Reply:
I had that thought at one point too, and I think it might be nice to do. We could also add a placeholder animation of some sort to "spin" while the job is sitting idle.